cardano-node/src/Cardano/Node/Parsers.hs

```diff
@@ -43,6 +43,7 @@ nodeRunParser = do
   shelleyKESFile  <- optional parseKesKeyFilePath
   shelleyVRFFile  <- optional parseVrfKeyFilePath
   shelleyCertFile <- optional parseOperationalCertFilePath
+  shelleyBulkCredsFile <- optional parseBulkCredsFilePath
```

Comment:
We need a better commit message on this. I am **TOTALLY** uninterested in the files that conflicted in some merge.

The commit message **should** contain information about what this commit actually does and referring to a Jira ticket that people outside IOHK cannot view.